src/UI/Components/home_header/index.tsx

```diff
@@ -0,0 +1,28 @@
+
+import { View, Text, Image, StyleSheet, ImageSourcePropType } from 'react-native';
+import AvatarCircle from '../AvatarCircle';
+import { styles } from './styles';
+
+interface HeaderProps {
+  username: string;
+  profileImage: ImageSourcePropType;
+}
+
+const Header: React.FC<HeaderProps> = ({ username, profileImage }) => {
```

Comment:
We do not use React.FC component at all, just make an arrow function taking your type as props 
for eg 
`const Header = (props: HeaderProps)=>{...}`